RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,47 @@
+# React Native Paper Component Templates
+
+This Document contains a set of components Templates around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.
+
+By default, the theming mechanism provided by RNP is very minimal and does not allow us to style components precisely the way we want for Brightlayer UI applications. To circumvent this issue, we create a template that you can use bring them in alignment with Brightlayer UI applications
```

Comment:
By default, the theming mechanism provided by React Native Paper is very minimal and does not allow us to style components precisely the way we want for Brightlayer UI applications. To circumvent this issue, we create a template that you can use to bring them in alignment with Brightlayer UI Design System and Applications.

Reply:
just suggestion see what others say

Reply:
I would put this as the first paragraph. In addition, this is also part of the reason why we offer these theme overrides:

> Since React Native Paper initially released v5 in TIME, Material Design has made numerous updates. As a result, the default style and theme structure offered by React Native Paper is outdated.